### PR TITLE
docs: Update resource table format

### PR DESCRIPTION
**Pull request by moduli**
This PR is a continuation of https://github.com/hashicorp/boundary/pull/3141.

The above PR intended to modify the format of the [Resource Table topic](https://developer.hashicorp.com/boundary/docs/concepts/security/permissions/resource-table), but in order to do so, we needed to update the scripts that generated that table. This PR updates those scripts to generate the table in the desired format.

[Vercel preview](https://boundary-ovsvua1yf-hashicorp.vercel.app/boundary/docs/concepts/security/permissions/resource-table)

### CODE REVIEW

File: website/content/docs/concepts/security/permissions/resource-table.mdx [hunk 1]
```diff
@@ -5,1738 +5,179 @@ description: |-
   Resource table
 ---
 
-# Resource table
+# Resource tables
 
-The following table works as a quick cheat-sheet to help you manage your
-permissions. Note that it's not exhaustive; for brevity it does _not_ show
+The following tables work as a quick cheat-sheet to help you manage your
```
Comment (Dan-Heath) [inline website/content/docs/concepts/security/permissions/resource-table.mdx:10]: Good catch on the singular > plural!

Comment (moduli) [reply website/content/docs/concepts/security/permissions/resource-table.mdx:10]: Can't take any credit there. Those were on your original PR 😅

**Review by Dan-Heath — APPROVED** [review]
Looks great to me! Thanks for doing this.

**Review by stellarsquall — APPROVED** [review]
Huge improvement, thank you!

**Comment by stellarsquall** [comment]
@jefferai, would you mind taking a brief look before we merge this? Want to make sure you don't have any concerns.

**Review by jefferai — APPROVED** [review]

**Pull request merged**